src/Uno.UI.RuntimeTests/IntegrationTests/dxaml/controls/calendardatepicker/CalendarDatePickerIntegrationTests.cs

```diff
@@ -130,121 +176,87 @@ public async Task CanEnterAndLeaveLiveTree()
 		}
 
 		[TestMethod]
-		[Ignore("https://github.com/unoplatform/uno/issues/10165")]
+		[RunsOnUIThread]
 		public async Task CanOpenFlyoutByTapping()
```

Comment:
These tests originate from MS sources. We shouldn't be modifying them in most cases.